### ISSUE TITLE
Description flipped?

### ISSUE THREAD

**Issue by sjshim** [jmumford/rt_data_analysis#1]
https://github.com/jmumford/rt_data_analysis/blob/5bbad7a43ca1bb9eeae3d4deaba99f95a8ed3cd0/main_analysis_code/utils_lev1/qa.py#L50

Is this description flipped?
True=design is bad
False=design is good

**Comment by jmumford** [comment]
Yes, it looks like it is.  True = there are problems, False=No problems.
Thanks for catching that. It is implemented correctly in the code that uses
it and just the description is wrong.
